### ISSUE TITLE
v3.8.1 restart loop detected

### ISSUE THREAD

**Issue by Brainbug01** [iobroker-community-adapters/ioBroker.telegram#840]
Version 3.8.1

Log from start to crash:

```
2024-07-15 18:33:41.909 - info: host.iobroker iobroker host.iobroker Adapter "system.adapter.telegram.0" is started
2024-07-15 18:33:41.912 - info: host.iobroker iobroker host.iobroker Adapter "system.adapter.telegram.1" is started
2024-07-15 18:33:41.913 - info: host.iobroker "system.adapter.telegram.0" enabled
2024-07-15 18:33:41.943 - info: host.iobroker iobroker host.iobroker Adapter "system.adapter.telegram.2" is started
2024-07-15 18:33:41.944 - info: host.iobroker "system.adapter.telegram.1" enabled
2024-07-15 18:33:41.949 - info: host.iobroker "system.adapter.telegram.2" enabled
2024-07-15 18:33:42.223 - info: host.iobroker instance system.adapter.telegram.0 in version "3.8.1" started with pid 258426
2024-07-15 18:33:42.240 - info: host.iobroker instance system.adapter.telegram.2 in version "3.8.1" started with pid 258432
2024-07-15 18:33:42.245 - info: host.iobroker instance system.adapter.telegram.1 in version "3.8.1" started with pid 258434
2024-07-15 18:33:42.945 - info: host.iobroker iobroker exit 0
2024-07-15 18:33:43.231 - info: telegram.2 (258432) starting. Version 3.8.1 in /opt/iobroker/node_modules/iobroker.telegram, node: v20.15.1, js-controller: 6.0.8
2024-07-15 18:44:48.995 - error: telegram.0 (258426) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:44:48.995 - error: telegram.0 (258426) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:49.073 - error: telegram.0 (258426) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:49.073 - error: telegram.0 (258426) Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:49.581 - warn: telegram.0 (258426) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:44:50.117 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:44:50.118 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:50.118 - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:44:50.118 - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-15 18:44:51.255 - info: host.iobroker instance system.adapter.telegram.0 in version "3.8.1" started with pid 259311
2024-07-15 18:44:54.087 - error: telegram.2 (258432) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:44:54.087 - error: telegram.0 (259311) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:44:54.087 - error: telegram.1 (258434) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:44:54.088 - error: telegram.0 (259311) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.167 - error: telegram.0 (259311) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:54.168 - error: telegram.0 (259311) Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.087 - error: telegram.2 (258432) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.181 - error: telegram.2 (258432) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:54.181 - error: telegram.2 (258432) Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.088 - error: telegram.1 (258434) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.206 - error: telegram.1 (258434) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:54.206 - error: telegram.1 (258434) Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:54.679 - warn: telegram.0 (259311) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:44:54.690 - info: telegram.2 (258432) terminating
2024-07-15 18:44:54.691 - warn: telegram.2 (258432) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:44:54.715 - warn: telegram.1 (258434) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:44:55.208 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:44:55.208 - error: host.iobroker Caught by controller[0]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:55.208 - error: host.iobroker Caught by controller[0]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:44:55.208 - error: host.iobroker Caught by controller[0]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:44:55.208 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:44:55.209 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:44:55.209 - error: host.iobroker Caught by controller[0]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:44:55.209 - error: host.iobroker Caught by controller[0]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:44:55.209 - error: host.iobroker Caught by controller[0]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:55.209 - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:44:55.209 - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:44:55.228 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:55.228 - error: host.iobroker instance system.adapter.telegram.2 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:44:55.228 - info: host.iobroker Restart adapter system.adapter.telegram.2 because enabled
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:44:55.245 - error: host.iobroker Caught by controller[0]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:44:55.245 - error: host.iobroker instance system.adapter.telegram.1 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:44:55.245 - info: host.iobroker Restart adapter system.adapter.telegram.1 because enabled
2024-07-15 18:44:56.473 - info: host.iobroker instance system.adapter.telegram.0 in version "3.8.1" started with pid 259339
2024-07-15 18:44:56.482 - info: host.iobroker instance system.adapter.telegram.2 in version "3.8.1" started with pid 259345
2024-07-15 18:44:56.487 - info: host.iobroker instance system.adapter.telegram.1 in version "3.8.1" started with pid 259346
2024-07-15 18:44:57.502 - info: telegram.2 (259345) starting. Version 3.8.1 in /opt/iobroker/node_modules/iobroker.telegram, node: v20.15.1, js-controller: 6.0.8
2024-07-15 18:45:01.089 - error: telegram.0 (259339) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:45:01.153 - error: telegram.2 (259345) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:45:01.153 - error: telegram.1 (259346) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:45:01.089 - error: telegram.0 (259339) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.149 - error: telegram.0 (259339) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:01.149 - error: telegram.0 (259339) Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.154 - error: telegram.1 (259346) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.229 - error: telegram.1 (259346) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:01.229 - error: telegram.1 (259346) Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.153 - error: telegram.2 (259345) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.235 - error: telegram.2 (259345) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:01.235 - error: telegram.2 (259345) Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:01.658 - warn: telegram.0 (259339) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:45:01.737 - warn: telegram.1 (259346) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:45:01.743 - info: telegram.2 (259345) terminating
2024-07-15 18:45:01.744 - warn: telegram.2 (259345) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:45:02.191 - error: host.iobroker Caught by controller[0]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:02.191 - error: host.iobroker instance system.adapter.telegram.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:45:02.191 - info: host.iobroker Restart adapter system.adapter.telegram.0 because enabled
2024-07-15 18:45:02.191 - warn: host.iobroker Do not restart adapter system.adapter.telegram.0 because restart loop detected
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:45:02.266 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:02.266 - error: host.iobroker instance system.adapter.telegram.1 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:45:02.267 - info: host.iobroker Restart adapter system.adapter.telegram.1 because enabled
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:45:02.280 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:02.280 - error: host.iobroker instance system.adapter.telegram.2 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:45:02.280 - info: host.iobroker Restart adapter system.adapter.telegram.2 because enabled
2024-07-15 18:45:03.441 - info: host.iobroker instance system.adapter.telegram.1 in version "3.8.1" started with pid 259398
2024-07-15 18:45:03.446 - info: host.iobroker instance system.adapter.telegram.2 in version "3.8.1" started with pid 259399
2024-07-15 18:45:04.300 - info: telegram.2 (259399) starting. Version 3.8.1 in /opt/iobroker/node_modules/iobroker.telegram, node: v20.15.1, js-controller: 6.0.8
2024-07-15 18:45:06.737 - info: nuki.0 (41834) action 3 set successfully
2024-07-15 18:45:11.768 - info: nuki.0 (41834) status change received for NukiID 997884141: unlocked
2024-07-15 18:45:11.768 - info: nuki.0 (41834) battery status received for NukiID 997884141: false
2024-07-15 18:45:19.669 - error: telegram.2 (259399) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:45:19.669 - error: telegram.1 (259398) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-07-15 18:45:19.669 - error: telegram.2 (259399) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:19.734 - error: telegram.2 (259399) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:19.734 - error: telegram.2 (259399) Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:19.669 - error: telegram.1 (259398) unhandled promise rejection: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:19.748 - error: telegram.1 (259398) TypeError: Cannot read properties of undefined (reading 'split')
at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
at AdapterClass.emit (node:events:519:28)
at AdapterClass.emit (node:domain:488:12)
at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:19.748 - error: telegram.1 (259398) Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:20.240 - info: telegram.2 (259399) terminating
2024-07-15 18:45:20.241 - warn: telegram.2 (259399) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:45:20.256 - warn: telegram.1 (259398) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:45:20.774 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:20.774 - error: host.iobroker instance system.adapter.telegram.2 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:45:20.774 - info: host.iobroker Restart adapter system.adapter.telegram.2 because enabled
2024-07-15 18:45:20.775 - warn: host.iobroker Do not restart adapter system.adapter.telegram.2 because restart loop detected
2024-07-15 18:45:20.796 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-07-15 18:45:20.796 - error: host.iobroker Caught by controller[1]: TypeError: Cannot read properties of undefined (reading 'split')
2024-07-15 18:45:20.796 - error: host.iobroker Caught by controller[1]: at processMessage (/opt/iobroker/node_modules/iobroker.telegram/main.js:1191:52)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at AdapterClass. (/opt/iobroker/node_modules/iobroker.telegram/main.js:156:17)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:events:519:28)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at AdapterClass.emit (node:domain:488:12)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at change (/opt/iobroker/node_modules/@iobroker/js-controller-adapter/src/lib/adapter/adapter.ts:10949:34)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at Immediate. (file:///opt/iobroker/node_modules/@iobroker/db-states-redis/src/lib/states/statesInRedisClient.ts:383:37)
2024-07-15 18:45:20.797 - error: host.iobroker Caught by controller[1]: at processImmediate (node:internal/timers:478:21)
2024-07-15 18:45:20.797 - error: host.iobroker instance system.adapter.telegram.1 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-07-15 18:45:20.797 - info: host.iobroker Restart adapter system.adapter.telegram.1 because enabled
2024-07-15 18:45:20.797 - warn: host.iobroker Do not restart adapter system.adapter.telegram.1 because restart loop detected
```

 - Adapter version: 3.8.1
 - JS-Controller version: 6.0.8
 - Node version: 20.15.1
 - Operating system: Linux

**Comment by mcm1957** [comment]
@GermanBluefox 
Please check if this is related to your last changes

**Comment by solarsnoop** [comment]
same here : Node.js v18.17.1 
NPM: 9.6.7
JS : 6.0.8

3.8.0 running fine , upgrade to 3.8.1 loops same descripted
roll back to 3.8.0 all is working fine again

**Comment by mcm1957** [comment]
fixed with 3.8.2
Please test with 3.8.2

**Comment by solarsnoop** [comment]
ok thank you. Installed and will report in 1h
